src/components/AnArticle.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import ReactHtmlParser from 'react-html-parser';
+
+const ratings = (aRating) => {
+  const stars = [];
+  const aStar = (<FontAwesomeIcon icon="star" />);
+  let counter = 0;
+  while (counter < aRating) {
+    stars.push(aStar);
+    counter += 1;
+  }
```

Comment:
@Etomovich, why are you using a while loop here, this can be done using sorts or maps.

Reply:
I wanted to reduce the amount of code in the renderer but that can easily be refactored

Reply:
You can use a filter or map to achieve this.